pkg/agent/openflow/client_test.go

```diff
@@ -893,6 +894,17 @@ func Test_client_InstallServiceGroup(t *testing.T) {
 				"bucket=bucket_id:0,weight:100,actions=set_field:0xfec00010001000000000000000000100->xxreg3,set_field:0x50/0xffff->reg4,resubmit:ServiceLB," +
 				"bucket=bucket_id:1,weight:100,actions=set_field:0xfec00010001000000000000000000101->xxreg3,set_field:0x50/0xffff->reg4,resubmit:ServiceLB",
 		},
+		{
+			name: "IPv4 Endpoints",
```

Comment:
Rename this since there is a test called `IPv4 Endpoints`

Reply:
updated